README.md

```diff
@@ -52,22 +51,44 @@ yarn
 yarn run start
 ```
 
-
 ## Description
 
 This demo will assist you in seeing a Chime SDK Meeting media capture pipeline in action.  It uses a simple React based client to create a meeting and then start a media capture pipeline of that meeting.  This pipeline will take chunks of media in 5 second increments and store them in an S3 bucket.  Once the recording has been stopped, a separate API call will trigger the Process Lambda and concatenate the separate files together into a single, playable media file.  
 
-This demo relies on ffmpeg for concatenating the files together.  An ffmpeg layer based on [John Van Sickle's static build](https://johnvansickle.com/ffmpeg/) is deployed as part of this deployment.  For licensing [information](http://ffmpeg.org/legal.html).
+This demo relies on ffmpeg for concatenating the files together.  An Docker image is created that uses [John Van Sickle's static build](https://johnvansickle.com/ffmpeg/) and is deployed as part of this deployment.  For licensing [information](http://ffmpeg.org/legal.html).
```

Comment:
Typo:`A Docker image` (instead of An)

Reply:
Hi, 

We're able to record and stored the audio files to the S3 bucket. As we tried to implement the process Lambda which is in python, we encountered an error. We added the function in the lambda function using python runtime.  We tried to add the ffpmeg package as well to the lambda layer.

Any idea what causes the issue?

CompletedProcess(args=['ffmpeg', '-i', '/tmp/2021-09-28-05-46-39-026.mp4', '-bsf:v', 'h264_mp4toannexb', '-f', 'mpegts', '-framerate', '15', '-c', 'copy', '/tmp/2021-09-28-05-46-39-026.ts', '-y'], returncode=127, stdout=b'', stderr=b'-i: ffmpeg: command not found\n')

Thank you so much in advance for your response.

Thank you.